### PR TITLE
create a new branch for auto-tuning pr

### PR DESCRIPTION
**Pull request by yintong-lu**
## Type of Change

feature update

## Description

introduced a new method to perform alpha auto-tuning in smooth_quant

## Expected Behavior & Potential Risk

the expected behavior that triggered by this PR 

## How has this PR been tested?

ut and llm validation

## Dependency Change?

no

### CODE REVIEW

File: neural_compressor/adaptor/torch_utils/smooth_quant.py [hunk 1]
```diff
@@ -82,16 +94,20 @@ def model_forward_per_sample(model, sample, device):
         return output
 
 
-def quant_dequant_w(m, num_bits=8, scheme='asym'):  ##TODO take sym as default
+def quant_dequant_w(m, num_bits=8, scheme='sym'):  ##TODO take sym as default
```
Comment (lkk12014402) [inline neural_compressor/adaptor/torch_utils/smooth_quant.py:97]: remove comments.

**Review by lkk12014402 — APPROVED** [review]

**Review by n1ck-guo — APPROVED** [review]

**Pull request merged**